<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Music_Albums_Add</class>
 <widget class="QDialog" name="Music_Albums_Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Album</string>
  </property>
  <widget class="QLabel" name="AddAlbum_NumberText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Album Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddAlbum_TitleBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_TitleText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Album Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_DateText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Release Date</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="AddAlbum_DateBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddAlbum_KeyWordsBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_KeyWordsText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Key Words</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddAlbum_CommentsBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_CommentsText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="AddAlbum_NumberBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddAlbum_CodeBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_CodeText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Authorisation Code</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddAlbum_CodeButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_MainText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add New Album</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="AddAlbum_CancelButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_TitleValidation">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_KeyValidation">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_CommentsValidation">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_CodeValidation">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AddAlbum_CodeButton</sender>
   <signal>clicked()</signal>
   <receiver>Music_Albums_Add</receiver>
   <slot>AddAlbum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddAlbum_CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Music_Albums_Add</receiver>
   <slot>CancelButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddAlbum()</slot>
  <slot>CancelButton()</slot>
 </slots>
</ui>
